lib/core/diagnostics.js

```diff
@@ -114,6 +122,75 @@ if (undiciDebugLog.enabled || fetchDebuglog.enabled) {
   isClientSet = true
 }
 
+// Track fetch requests
+if (fetchDebuglog.enabled && diagnosticsChannel.tracingChannel) {
```

Comment:
Shouldn't this code be placed by users instead of having a helper inside undici core?